src/main/scala/org/hammerlab/guacamole/commands/GermlineAssemblyCaller.scala

```diff
@@ -118,7 +121,7 @@ object GermlineAssemblyCaller {
           initialState = None,
           (lastCalledLocus, windows) => {
             val window = windows.head
-            val referenceName = window.contigName
```

Comment:
starting to standardize around `contigName` for this kind of variable; open to changing it, but a good first step is homogenizing uses.